### ISSUE TITLE
idris2-lsp does not build with latest main branch of idris 2

### ISSUE THREAD

**Issue by thomasdziedzic** [idris-community/idris2-lsp#108]
I believe this PR broke idris2-lsp building with the latest main: https://github.com/idris-lang/Idris2/pull/2076

```
61/68: Building Language.LSP.SignatureHelp (src/Language/LSP/SignatureHelp.idr)
Error: While processing left hand side of findRefNamesInTerm. When unifying:
    Name -> Term ?vars
and:
    Term ?_
Mismatch between: Name -> Term ?vars and Term ?_.

Language.LSP.SignatureHelp:46:21--46:29
 42 | findRefNamesInTerm (TDelay fc x ty arg)   = findRefNamesInTerm ty
 43 | findRefNamesInTerm (TForce fc x y)        = findRefNamesInTerm y
 44 | findRefNamesInTerm (PrimVal fc c)         = []
 45 | findRefNamesInTerm (Erased fc imp)        = []
 46 | findRefNamesInTerm (TType fc)             = []
                          ^^^^^^^^

make: *** [Makefile:11: build] Error 1
```

**Comment by ShinKage** [comment]
Closed by #109.

**Comment by thomasdziedzic** [comment]
@ShinKage thank you! can confirm it compiles now
